workloads/templates/common-templates.md

```diff
@@ -53,3 +53,46 @@ This make it possible to query for all the VMs built from any template.
 
 Please note that, after the generation step, VM objects and template objects have no relationship with each other besides the aforementioned label (e.g. changes
 in templates do not automatically affect VMs, or vice versa).
+
+## common template customization
+
+### template customization - memory, CPU
+
+There are three options to customize VM memory and CPU:
+
+ * select flavor - tiny, small, medium, etc. Each flavor grants to VM different amount of RAM and CPU cores.
+ * setting up directly editable fields spec.template.spec.domain.cpu.cores and spec.template.spec.domain.resources.requests.memory, example:
+
+```bash
+oc patch virtualmachine testguest --type merge -p '{"spec":{"template":{"spec":{"domain":{"cpu":{"cores":'3'}}}}}}'
+virtualmachine.kubevirt.io/testguest patched
+```
+ * use WebUI: WebUI supports different flavors and Workload Profiles. Also it supports custom amount of memory and CPUs
+
+### template customization - networking
+
+There are two options to customize Networking in VM:
+ * setting up directly editable field(see example above): spec.template.spec.networks. You can edit following:
+   * name (of interface)
+   * pod
+ * edit networking in webUI. Please note: WebUI also edits spec.domain.interfaces and supports changing mac address
```

Comment:
AFAIK we can do that from the commandline too.

Reply:
...we should check if it is indeed the case, and amend the doc accordingly.

Reply:
I mean exactly cli.

Reply:
What do you suggest to change in the text

Reply:
Let me check, let's keep as it is meanwhile

Reply:
On second thought let's drop my comment.